test/uk/gov/hmrc/plasticpackagingtax/registration/config/AppConfigSpec.scala

```diff
@@ -122,5 +124,21 @@ class AppConfigSpec extends AnyWordSpec with Matchers with MockitoSugar {
         "http://localhost:9250/contact/beta-feedback-unauthenticated?service=plastic-packaging-tax"
       )
     }
+    "inspect feature flags" when {
+      "and check that 'liabilityPreLaunch' is true" in {
+        validAppConfig.isPreLaunch mustBe false
+      }
+    }
+  }
+
+  "with an empty config" should {
+
+    val emptyAppConfig: AppConfig = appConfig(Configuration(emptyConfig))
+
+    "inspect feature flags" when {
+      "and check that 'liabilityPreLaunch' default value is 'true'" in {
```

Comment:
Should the wording here be that we're checking that the flag is `false` by default rather than `true`?

Reply:
:facepalm:  yes  have change the default value but did not bother to read the test

Reply:
This has been addressed